### ISSUE TITLE
ES6 all the things

### ISSUE THREAD

**Issue by georgehrke** [owncloud/calendar#411]
currently blocked by lacking support in Safari
https://kangax.github.io/compat-table/es6/#safari9

**Comment by tcitworld** [comment]
Since babel is now used to support Safari, can we now start to use ES6 ?

**Comment by georgehrke** [comment]
Yes, I think that should be fine :)

**Comment by tcitworld** [comment]
Also, why do we keep the public/app.js in git ? It brings an awful lot of issues when rebasing a branch on another. 

People that install the app from git should know that they have to execute grunt to build the file.

**Comment by georgehrke** [comment]
Yes, I thought about yesterday as well.
We should get rid of it.

> People that install the app from git should know that they have to execute grunt to build the file.

People shouldn't use git for production anyway ^^

I will prepare a pull request

**Comment by tcitworld** [comment]
Thanks

> People shouldn't use git for production anyway ^^

Not for production, but to test it for instance. Anyway, maybe add a word about it inside the README.

**Comment by georgehrke** [comment]
We added babel for backward compatibility to Safari 9.
All new code merged into the calendar should be ES6.
Moving all code will be done over time.
